spec/test_files/VERSIONS.md

```diff
@@ -0,0 +1,8 @@
+| Version | iOS version | Android version | Common files version |
+|---------|-------------|-----------------|----------------------|
+| 4.5.3   | 4.15.2      | 5.6.5           | 4.5.2                |
+| 4.5.2   | 4.15.2      | 5.6.5           | 4.5.1                |
```

Comment:
I think @tonidero mentioned something about Unity having a different number of columns so might be nice to have a Unity style table in the test (and maybe one that is just super messed up to test handling of errors gracefully) 🤷‍♂️

Reply:
Added some extra tests